### ISSUE TITLE
macOS compatibility broken by 3.14

### ISSUE THREAD

**Issue by ilovezfs** [schweikert/fping#109]
https://bot.brew.sh/job/Homebrew%20Core/14403/version=sierra/console
```
==> make install
Making install in doc
make[2]: Nothing to be done for `install-exec-am'.
 .././install-sh -c -d '/usr/local/Cellar/fping/3.14/share/man/man8'
 /usr/bin/install -c -m 644 fping.8 fping6.8 '/usr/local/Cellar/fping/3.14/share/man/man8'
Making install in src
clang -DHAVE_CONFIG_H -I. -I..    -Wall -Wextra -Wno-sign-compare -g -O2 -c -o fping.o fping.c
clang -DHAVE_CONFIG_H -I. -I..    -Wall -Wextra -Wno-sign-compare -g -O2 -c -o seqmap.o seqmap.c
clang -DHAVE_CONFIG_H -I. -I..    -Wall -Wextra -Wno-sign-compare -g -O2 -c -o socket.o socket.c
clang -DHAVE_CONFIG_H -I. -I..    -Wall -Wextra -Wno-sign-compare -g -O2 -c -o socket4.o socket4.c
clang -DHAVE_CONFIG_H -I. -I..    -Wall -Wextra -Wno-sign-compare -DIPV6 -g -O2 -c -o fping6-fping.o `test -f 'fping.c' || echo './'`fping.c
fping.c:391:23: error: use of undeclared identifier 'IP_PMTUDISC_DO'
            int val = IP_PMTUDISC_DO;
                      ^
fping.c:392:43: error: use of undeclared identifier 'IP_MTU_DISCOVER'
            if (setsockopt(s, IPPROTO_IP, IP_MTU_DISCOVER, &val, sizeof(val))) {
                                          ^
clang -DHAVE_CONFIG_H -I. -I..    -Wall -Wextra -Wno-sign-compare -DIPV6 -g -O2 -c -o fping6-seqmap.o `test -f 'seqmap.c' || echo './'`seqmap.c
2 errors generated.
make[1]: *** [fping.o] Error 1
make[1]: *** Waiting for unfinished jobs....
fping.c:391:23: error: use of undeclared identifier 'IP_PMTUDISC_DO'
            int val = IP_PMTUDISC_DO;
                      ^
fping.c:392:43: error: use of undeclared identifier 'IP_MTU_DISCOVER'
            if (setsockopt(s, IPPROTO_IP, IP_MTU_DISCOVER, &val, sizeof(val))) {
                                          ^
2 errors generated.
make[1]: *** [fping6-fping.o] Error 1
make: *** [install-recursive] Error 1
```

**Comment by ilovezfs** [comment]
Thanks!

**Comment by schweikert** [comment]
No, thanks to you for reporting this so quickly!

**Comment by ilovezfs** [comment]
@schweikert you're welcome. Patch applied: https://github.com/Homebrew/homebrew-core/pull/8694/files

**Comment by xtaran** [comment]
JFTR: This issue also seems to affect GNU/Hurd. At least [fping 3.14 no more compiles on Debian GNU/Hurd either](https://buildd.debian.org/status/logs.php?pkg=fping&arch=hurd-i386).

David: Do you plan a bug fix release because of that or shall I rather apply that patch in Debian, too?

**Comment by xtaran** [comment]
Applied it to the Debian package, too. It indeed helped and the package built fine again on Debian GNU/Hurd and Debian GNU/kFreeBSD (32-bit at least, amd64 still pending) again.

**Comment by schweikert** [comment]
released v3.15 with the fix
